cmd/cmd.go

```diff
@@ -85,9 +85,13 @@ func addOutputFlag(flags *pflag.FlagSet) {
 		"directory where downloaded files are put")
 }
 
-func addFilterFlag(flags *pflag.FlagSet) {
+func addFilterFlag(flags *pflag.FlagSet, defaultFilter ...string) {
```

Comment:
What's the point of passing multiple default filters to this function if only the first one is used?

Reply:
Not needed anymore, removed in 92f8a89ddf28f4295f403dc802cb18254fecc136